Tests/CouchDBTests/DocumentCrudTests.swift

```diff
@@ -44,10 +46,20 @@ class DocumentCrudTests: CouchDBTest {
                                     created_at: "Mon Aug 27 20:16:20 +0000 2012",
                                     favorited: false,
                                     value: "value2")
+    let myDocument4 = TypeADocument(_id: "654321+",
```

Comment:
Could you replace the literal ID string here with `documentId4` instead? (avoid duplicating strings) - same comment goes for all the documents defined here